hack/go-fmt.sh

```diff
@@ -1,19 +1,7 @@
 #!/bin/sh
 
-REPO_NAME=$(basename "${PWD}")
-if [ "$IS_CONTAINER" != "" ]; then
-  for TARGET in "${@}"; do
-    find "${TARGET}" -name '*.go' ! -path '*/vendor/*' ! -path '*/.build/*' -exec gofmt -s -w {} \+
-  done
-  git diff --exit-code
-else
-  docker run -it --rm \
```

Comment:
Is this now covered by the makefile?

Reply:
All of these above are covered by Makefile. Also checked if it does not influence our CI, and it should not. 
Also we should probably remove these [`IS_CONTAINER=TRUE`,](https://github.com/openshift/release/blob/master/ci-operator/config/openshift/cluster-api-provider-aws/openshift-cluster-api-provider-aws-master.yaml#L50) since the does nothing after this PR.